sale_order_display_country/models/sale_order.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    country = fields.Many2one(
+        related="partner_id.country_id", readonly=True, store=True,
+    )
+    country_code = fields.Char(related="partner_id.country_id.code")
```

Comment:
```suggestion
    country_code = fields.Char(related="partner_id.country_id.code", store=True)
```
May be worth it to store the value for performance reason if the filter is to be used frequently.